compliance/check.py

```diff
@@ -62,8 +62,7 @@ class CheckerWarning(Exception):
     TESTING_MODE + "/spl.txt",
 ]
 
-COMMON_ERROR = "Can't evaluate uncertainty of this sample!"
-
+COMMON_ERROR = ["Can't evaluate uncertainty of this sample!","USB."]
```

Comment:
Can we break this into `COMMON_ERROR_RANGING` and `COMMON_ERROR_TESTING`? The `"Can't evaluate uncertainty of this sample!"` error is only allowed in ranging phase, whereas `"USB."` should be allowed in both ranging and testing phases.